drivers/usb/dwc2/platform.c

```diff
@@ -468,6 +471,31 @@ static int dwc2_driver_probe(struct platform_device *dev)
 	if (retval)
 		return retval;
 
```

Comment:
Let's add "#ifdef CONFIG_MACH_STM32H743" here to avoid extra code on other platforms